client/jetpack-cloud/sections/partner-portal/components/credit-card-entry-form/index.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement, useCallback, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import page from 'page';
+import {
+	StripeHookProvider,
+	useStripe,
+	createStripePaymentMethod,
+} from '@automattic/calypso-stripe';
+import { CardElement } from 'react-stripe-elements';
+
+/**
+ * Internal dependencies
+ */
+import FormTextInput from 'calypso/components/forms/form-text-input';
+import FormTelInput from 'calypso/components/forms/form-tel-input';
+import FormLabel from 'calypso/components/forms/form-label';
+import { Button, Card } from '@automattic/components';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function CreditCardEntryForm( props ): ReactElement {
+	const translate = useTranslate();
+	const { isStripeLoading, stripeLoadingError, stripeConfiguration, stripe } = useStripe();
+
+	const [ cardName, setCardName ] = useState( '' );
+	const [ cardEmail, setCardEmail ] = useState( '' );
+	const [ cardPhone, setCardPhone ] = useState( '' );
+
+	const handleSubmit = ( e ) => {
+		e.preventDefault();
+	};
+
+	return (
+		<div className="credit-card-entry-form">
+			<div className="credit-card-entry-form__header">{ translate( 'Credit card details' ) }</div>
+			<div className="credit-card-entry-form__field">
+				<FormLabel>{ translate( 'Name' ) }</FormLabel>
+				<FormTextInput placeholder="Jane Doe" />
+			</div>
+			<div className="credit-card-entry-form__field">
+				<FormLabel>{ translate( 'Email' ) }</FormLabel>
+				<FormTextInput placeholder="janedoe@gmail.com" />
+			</div>
+			<div className="credit-card-entry-form__field">
+				<FormLabel>{ translate( 'Phone' ) }</FormLabel>
+				<FormTelInput placeholder="(951) 555-0123" />
+			</div>
+			<div className="credit-card-entry-form__field">
+				<FormLabel>{ translate( 'Card details' ) }</FormLabel>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Edit Card Details', { context: 'Header text'} )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-partner-payment)